addons/udes_sale_stock/models/stock_move.py

```diff
@@ -9,6 +9,11 @@ class StockMove(models.Model):
 
     _inherit = 'stock.move'
 
+    u_is_privacy_wrapped = fields.Boolean(
+        string='Is privacy wrapped',
+        compute='_compute_privacy_wrapped',
```

Comment:
Another option is to use a related field, with readonly=True, related='sale_line_id.privacy_wrapping' ? Although it is not going to be a boolean field.

Reply:
I like that better, I'll ask what would be preferred tomorrow. :)